### PR TITLE
Policy studio - Impl save button  

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**
https://gravitee.atlassian.net/browse/APIM-1550

**Description**

Add Output to send whit changes to save when the user Save

**Additional context**

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-policy-studio-angular
```
npm install @gravitee/ui-policy-studio-angular@7.4.0-apim-1550-save-output-7fb96a0
```
```
yarn add @gravitee/ui-policy-studio-angular@7.4.0-apim-1550-save-output-7fb96a0
```

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-particles-angular
```
npm install @gravitee/ui-particles-angular@7.4.0-apim-1550-save-output-7fb96a0
```
```
yarn add @gravitee/ui-particles-angular@7.4.0-apim-1550-save-output-7fb96a0
```

---

### CODE REVIEW

File: ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts [hunk 1]
```diff
@@ -37,9 +39,21 @@ export default {
...
     [plans]="plans"
+    (commonFlowsChange)="commonFlowsChange($event)"
+    (plansToUpdate)="plansToUpdate($event)"
     >
     </gio-policy-studio></div>`,
-    props,
+    props: {
+      ...props,
+      commonFlowsChange: (event: Flow[]) => {
+        console.info('commonFlowsChange', event);
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts:49]: Is console necessary if you are also creating an action?

File: ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.component.html [hunk 2]
```diff
@@ -33,7 +33,7 @@
   </div>
 
   <div class="header__btn">
-    <button mat-stroked-button color="primary">Save</button>
+    <button mat-stroked-button color="primary" [disabled]="!enableSaveButton" (click)="onSave()">Save</button>
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.component.html:36]: What about naming it `disableSaveButton` and reversing the logic?

**Review by jourdiw — APPROVED** [review]

File: ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/gio-ps-flow-form-dialog.component.spec.ts [hunk 3]
```diff
@@ -90,6 +90,7 @@ describe('GioPolicyStudioFlowFormDialogComponent', () => {
   it('should return false on cancel', async () => {
     component.flow = {
       _id: 'test-id',
+      _asChanged: false,
```
Comment (loriepisicchio) [inline ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/gio-ps-flow-form-dialog.component.spec.ts:93]: suggestion: rename to `_hasChanged`

**Review by loriepisicchio — APPROVED** [review]

File: ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts (hunk 1 above)
Comment (ThibaudAV) [reply ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts:49]: Yes sometime i use it to debug 🙈

**Pull request merged**